pom.xml

```diff
@@ -183,7 +183,7 @@
         <!-- Library Dependency Versions -->
         <org.osgi.service.http.port>8080</org.osgi.service.http.port>
         <org.osgi.service.http.port.secure>8443</org.osgi.service.http.port.secure>
-        <jackson.version>2.3.2</jackson.version>
+        <jackson.version>2.5.0</jackson.version>
```

Comment:
iron.io has version 2.5.0 in its pom.  and it causes a version conflict when parsing xml